docs/config.rst

```diff
@@ -17,7 +17,8 @@ Example:
         "icons": "icons",
         "styles": "styles",
         "mbtiles": "data",
-        "pmtiles": "data"
+        "pmtiles": "data",
+        "files": "data"
```

Comment:
This should proably be 
`"files": "files"` 
just so it doesn't default to sharing your full data directory.

Reply:
For data exposure I had the same thoughts after committing, for my default setup I use "/files", but I think a default beneath "public" will be a good indication, that these files will be exposed!

Reply:
Changed to "public/files" per default.